backend/routes/router.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
```

Comment:
optional
I don't see express besing used anywhere in the code. I would consider removing it if its not used